docs/installation.rst

```diff
@@ -37,6 +37,9 @@ On a Windows and Mac operating systems, standalone desktop software ``explorepy-
 `release page <https://github.com/Mentalab-hub/explore-desktop-release/releases/latest/>`_. Please note that the dependencies will be installed automatically.
 
 
+A standalone Mac installer is also available. Please contact Mentalab support to get Mac installer.
```

Comment:
The standalone Mac installer is available on the release page linked, the "contact support" line is not necessary anymore, in my opinion (but a link to the GUI wiki might be useful?)